resources/web/style/conum.pcss

```diff
@@ -6,7 +6,6 @@
     line-height: 16px;
     color: #f5f7fa;
     background-color: #0080d5;
-    -webkit-border-radius: 100px;
```

Comment:
Autoprefixer doesn't handle this one because it hasn't been needed in a long time.